app/models/quote.rb

```diff
@@ -0,0 +1,99 @@
+class Quote < ActiveRecord::Base
+
+  def self.get_rates(request)
+    parsed_request = JSON.parse(request)
+    @quote = self.new
+    @quote.request = request
+    @origin = Quote.get_origin
+    @packages = Quote.get_packages(parsed_request["number_of_items"])
```

Comment:
so maybe I've forgotten how everything works in ruby and rails- looks like you don't use these @quote, @quote.request and @packages in the controller, so you can just leave the @ out of there. I think.